resource/StandardResources.md

```diff
@@ -0,0 +1,105 @@
+# Standard Resources
+
+This page lists the standard resource types in OpenCensus. For more details on how resources can 
+be combined see [this](Resource.md)
+
+ECS defines these fields.
+ * [Compute Unit](#compute-unit)
+   * [container](#container)
+ * [Deployment Service](#deployment-service)
+   * [Kubernetes](#kubernetes)
+ * [Compute Instance](#compute-instance)
+   * [Host](#host)
+ * [Environment](#event)
+   * [Cloud](#cloud)
+   * [Cluster](#cluster)
+   * [Data Center](#data-center)
+
+## TODOs
+* Add logical compute units: Service, Task - instance running in a service.
+* Add more compute units: Process, Lambda Function, AppEngine unit, etc.
+
+## Compute Unit
+Resources defining a compute unit (e.g. Container, Process, Lambda Function).
+
+### Container
+**type:** `container`
+
+**Description:** A container instance. This resource can be merged with a deployment service
+resource, a compute instance resource and an environment resource.
+
+| Label  | Description  | Example  |
+|---|---|---|
+| container.id | Unique container id. |  |
+| container.image.name | Name of the image the container was built on. |  |
+| container.image.tag | Container image tag. |  |
+| container.name | Container name. |  |
+
+## Deployment Service
+Resources defining a deployment service (e.g. Kubernetes).
+
+### Kubernetes
+**type:** `k8s`
+
+**Description:** A Kubernetes pod resource. This resource can be merged with a compute instance
+resource and/or an environment resource.
+
+| Label  | Description  | Example  |
+|---|---|---|
+| k8s.cluster.name | The name of the cluster that the container is running in. |  |
+| k8s.namespace.name | The name of the namespace that the container is running in. |  |
+| k8s.pod.name | The name of the pod that the container is running in. |  |
+
+## Compute Instance
+Resources defining a computing instance (e.g. host).
+
+### Host
+**type:** `host`
+
+**Description:** A host is defined as a general computing instance.
+
+| Label  | Description  | Example  |
+|---|---|---|
+| host.hostname | Hostname of the host.<br/> |  |
+| host.id | Unique host id.<br/> For Cloud this must be the instance_id assigned by the cloud provider | `i-1234567890abcdef0` |
+| host.name | Hostname of the host. |  |
+| host.ip | Host ip address. |  |
```

Comment:
Would be helpful to add some clarification on IP address:
1. If there are both private and public IP address, which one should be used? (I guess the public one?)
2. For hosts with multiple IP addresses (e.g. both IPv4 and IPv6, or two IPv4), what's the recommendation.
3. Is this IPv4 only?

Reply:
Actually not 100% sure what to do with this. Maybe IP is not that important to have it right now and we should remove until we know exactly how to use this?

Reply:
Probably yes.